.gitignore

```diff
@@ -58,3 +58,6 @@ ChangeLog
 
 # vim
 *.swp
+
+# Intellij
+.idea
```

Comment:
Pro tip: you can use global `gitignore` files by configuring `core.excludesfile`, e.g. `git config --global core.excludesFile ~/.gitignore`. Avoids having to do this for every project you contribute to...

Reply:
cool, thanks for the tip and the rapid approval :D